package.json

```
@@ -10,7 +10,7 @@
     "services:down": "docker compose -f infra/compose.yaml down",
     "lint:check": "prettier --check .",
     "lint:fix": "prettier --write .",
-    "test": "jest --runInBand",
+    "test": "npm run services:up && npm run wait-for-postgres && concurrently -n next,jest --hide next -k -s command-jest 'next dev' 'jest --runInBand'",
```

Lembrei das aspas simples que vão gerar problemas dependendo do sistema operacional. Não tinha notado quando fiz a primeira revisão, mas apenas ao assistir a aula de hoje.

Melhor usar `\"`, como no PR https://github.com/filipedeschamps/tabnews.com.br/pull/592

```suggestion
    "test": "npm run services:up && npm run wait-for-postgres && concurrently -n next,jest --hide next -k -s command-jest \"next dev\" \"jest --runInBand\"",
```

---

Eu usei wait-on  [wait-on](https://www.npmjs.com/package/wait-on), no meu caso resolveu.
`"test": "npm run services:up && npm run wait-for-postgres && concurrently -n next,jest -k -s command-jest 'next dev' 'wait-on http://localhost:3000 && jest --runInBand'"`